### PR TITLE
feat: openstack deploy module update

### PR DESCRIPTION
**Pull request by jftalta**

### CODE REVIEW

File: cours/openstack/openstack-presentation-inside.fr.md [hunk 1]
```diff
@@ -151,14 +151,14 @@
 
 - Depuis 2017
```
Comment (rguichard) [inline cours/openstack/openstack-presentation-inside.fr.md:152]: s/2017/2013/ ?

**Review by rguichard — COMMENTED** [review]
`cours/openstack/openstack-deploiement-generalites.fr.md` is very close to `cours/openstack/openstack-deploiement.fr.md` but smaller. Should one replace the other ?

File: cours/openstack/openstack-presentation-inside.fr.md (hunk 1 above)
Comment (jftalta) [reply cours/openstack/openstack-presentation-inside.fr.md:152]: No. 2017 is right, according to this report : https://object-storage-ca-ymq-1.vexxhost.net/swift/v1/6e4619c416ff4bd19e1c087f27a43eea/www-assets-prod/reports/OpenStack-AnnualReport2017.pdf

**Comment by jftalta** [comment]
> `cours/openstack/openstack-deploiement-generalites.fr.md` is very close to `cours/openstack/openstack-deploiement.fr.md` but smaller. Should one replace the other ?

Yes, you're right. I will discard the smallest.

**Review by rguichard — APPROVED** [review]
lgtm

**Pull request merged**